internal/app/machined/pkg/controllers/runtime/kernel_param_config.go

```diff
@@ -0,0 +1,105 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I'm a very big fan of unit-tests for controllers. They seem to be trivial, but they allow us to catch bugs easily and sometimes allow to test features we rarely touch.

For example, I don't think we ever touch sysctls in our tests, so any regression might go unnoticed.